index.html

```diff
@@ -0,0 +1,231 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link href="https://cdn.jsdelivr.net/npm/bootstrap@5.0.2/dist/css/bootstrap.min.css" rel="stylesheet" integrity="sha384-EVSTQN3/azprG1Anm3QDgpJLIm9Nao0Yz1ztcQTwFspd3yD65VohhpuuCOmLASjC" crossorigin="anonymous">
+    <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/6.4.0/css/all.min.css" integrity="sha512-iecdLmaskl7CVkqkXNQ/ZH/XLlvWZOJyj7Yy7tcenmpD1ypASozpmT/E0iPtmFIB46ZmdtAc9eNBvH0H/ZpiBw==" crossorigin="anonymous" referrerpolicy="no-referrer"/>
+    <link rel="icon" type="image/png" sizes="32x32" href="./images/favicon-32x32.png">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+    <link href="https://fonts.googleapis.com/css2?family=Epilogue" rel="stylesheet">
+    <title>Frontend Mentor | Intro section with dropdown navigation</title>
+    <link rel="stylesheet" href="dist/assets/style/style.css">
+
+    <script src="https://code.jquery.com/jquery-3.7.0.min.js" integrity="sha256-2Pmvv0kuTBOenSvLm6bvfBSSHrUJ+3A7x6P5Ebd07/g=" crossorigin="anonymous"></script>
+    <script src="dist/assets/js/jquery.min.js "></script>
+</head>
+<body>
+<div class="container">
+    <div class="header">
+        <nav class="navbar navbar-expand-lg navbar-light d-flex align-items-md-start">
+            <h1 class="header-text">snap</h1>
+
+            <button class="navbar-toggler" type="button">
+                <span class="navbar-open"></span>
+            </button>
+
+            <div class="offcanvas offcanvas-end">
+                <div class="offcanvas-body">
+                    <div class="collapse navbar-collapse flex-column align-items-start show flex-md-row justify-content-md-between">
+                        <ul class="navbar-nav d-flex gap-2">
+                            <li class="nav-item dropdown">
+                                <a class="nav-link nav-rotate-one d-flex flex-direction-row align-items-center gap-2" href="#">
+                                    Features
+                                    <i class="fa-solid fa-angle-down fa-xs rotate-one"></i>
+                                </a>
+
+                                <ul class="dropdown-menu dropdown-list">
+                                    <li>
+                                        <a class="dropdown-item d-flex gap-3" href="#">
```

Comment:
Using gap to handle spacing or using pseudo element to remove padding of list item for last item to be mach a design